.github/shared/src/path.js

```diff
@@ -1,13 +1,37 @@
 // @ts-check
 
-import { resolve, sep } from "path";
+import { basename, dirname, resolve } from "path";
```

Comment:
Safer to use path functions `basename()` and `dirname()` compared to `string.split(sep)`.